wine/tricks.go

```diff
@@ -2,10 +2,34 @@ package wine
 
 import (
 	"log"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/vinegarhq/vinegar/internal/config"
 )
 
+func isWinerootTricksAvaliable(wineroot string) (bool, error) {
+	_, err := exec.LookPath(filepath.Join(wineroot, "bin", "winetricks"))
+	if err != nil {
+		return false, err
+	}
+	return true, nil
+
+}
+
 func (p *Prefix) DisableCrashDialogs() error {
 	log.Println("Disabling Crash dialogs")
 
 	return p.RegistryAdd("HKEY_CURRENT_USER\\Software\\Wine\\WineDbg", "ShowCrashDialog", REG_DWORD, "")
 }
+
+func (p *Prefix) LaunchWinetricks(cfg *config.Config) error {
+	log.Println("Launching winetricks")
+	result, err := isWinerootTricksAvaliable(cfg.WineRoot)
+	if err != nil && !result {
+		log.Println("winetricks not found in wineroot: " + err.Error() + ", falling back to system winetricks")
+		return p.Command("winetricks").Run()
+	}
+
+	return p.Command(filepath.Join(cfg.WineRoot, "bin", "winetricks")).Run()
+}
```

Comment:
```suggestion
func (p *Prefix) Winetricks() error {
	log.Println("Launching winetricks")
	
	return p.Command("winetricks").Run()
}
```
`winetricks` is a program. It is not part of a wine installation. and `PATH` is automatically appended to with the bin of wineroot when using the wineroot, how else would we be using the wineroot's wine?